### DISCUSSION TITLE
Aqara fp1 unable to set region or zone

### DISCUSSION THREAD

**Discussion by Emile86** [Koenkk/zigbee2mqtt#19223] [Devices]
I tried to find a solution after many trials i just give up.
Maybe someone will help me out.

I have successfully paird sensor but when i try to set any region or zone i get always errors 

`zigbee-herdsman-converters:xiaomi:aqara_fp1:region_upsert: encountered an error (ZONES_LIST_EMPTY) while parsing configuration commands (input: {"region_id":1})`

`zigbee-herdsman-converters:xiaomi:aqara_fp1:region_upsert: encountered an error (INVALID_REGION_ID) while parsing configuration commands (input: {"region_id":""})`

`zigbee-herdsman-converters:xiaomi:aqara_fp1:region_upsert: encountered an error (INVALID_REGION_ID) while parsing configuration commands (input: {"zones":[{"x":2}]})`

Do i miss something or i do something wrong?

Zigbee channel 11
Latest coordinator firmware

**Comment by sbvitok** [comment]
same problem
